packages/core/src/tracing/tracingHeaders.js

```diff
@@ -313,6 +348,30 @@ function readW3cTraceContext(headers) {
   return traceContext;
 }
 
+/**
+ * @typedef {Object} InstanaAncestor
+ * @property {string} t
+ * @property {string} p
+ */
+
+/**
+ * @typedef {Object} TraceResult
```

Comment:
*suggestion (non-blocking):*

This object represents the incoming headers (Instana headers and W3C trace context in a format that has already been processed and normalized and provides individual instrumentations (like httpServer or http2Server)  with the means to decide how to continue the trace and correlate it with a parent span.

I think we can find a better name than `TraceResult` -- that sounds like the result of tracing something, which is not accurate. We did not yet trace anything at this point. 

Better names might be one of:
- `TracingHeaders`,
- `TraceHeaders`,
- `TraceContext`.



WDYT?

Reply:
changed it to `TracingHeaders`